content/docs/operating/security.md

```diff
@@ -177,6 +181,24 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters. The projects will share
+the same TLS library, which will be based on the Golang vanilla TLS library. We
+keep [Golang's default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+explicitly disable TLSv1.0 and TLSv1.1.
```

Comment:
I feel this is worth re-discussing. At the time we didn't have TLSv1.3, I think we should default to that. Bonus points we get from that is that there are no configurable cipher suites with 1.3 yet anyways.

Reply:
That'd in addition to already breaking client_java and client_python compared to Go defaults, would also break client_golang (and possibly also client_ruby), plus I imagine a non-trivial amount of users.

Reply:
I would word this as "we support TLSv1.2 and higher".

Reply:
There is a difference between default to that and that only. I think that TLS1.3 is not spread enough to disable 1.2 entirely. Imagine that you run RHEL and you can't use curl to debug.

Reply:
"TLSv1.2 and higher" I'm happy with as well. Needs to be configurable then though.

Reply:
rephrased to use 1.2 and higher. The configuration bit I have not added as I see no consensus.

I think that we can discuss the configuration part at the next dev summit to reach a consensus.

Reply:
If you run RHEL when you need TLS 1.0 :)

Reply:
Yeah, configuration is something that is to be avoided - that's what a reverse proxy is for as it stands.

This doc still doesn't explain why we stick with defaults, or why we make an exception for 1.2. Defaults was for general sanity/maintainability, but a general basis for cherry-picking versions (and why only versions, not ciphers or the other 10+ settings) needs to be elucidated.